src/ImageSharp/Formats/Tiff/TiffEncoderPixelStorageMethod.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Formats.Experimental.Tiff
+{
+    /// <summary>
+    /// The tiff encoder pixel storage method.
+    /// </summary>
+    public enum TiffEncoderPixelStorageMethod
```

Comment:
I could not think of a better name, it can be renamed 😄

Reply:
to this enum possible to add tile, planar

Reply:
I would have thought whatever the enum is then it would be used for both decoder and encoder no? Would this not be part of the retrieved metadata?

Reply:
this is settings for decoder only, like [TiffEncodingMode](https://github.com/SixLabors/ImageSharp/blob/c57664dee01d7b527ee17347161484a53a4ee978/src/ImageSharp/Formats/Tiff/TiffEncodingMode.cs)

Reply:
it seems, this enum can be removed and always use multiStrip mode

Reply:
i would say remove this enum, and always use multiStrip, as its recommended by the spec anyway.

Reply:
Done 4895521f63395962d06555062292b79229f1e109

Reply:
> @IldarKhayrutdinov there seems to be an issue now with the `packedbits` compression. While the thumbnail of the tiffs look fine, if you open the image it looks broken. See for example in the Actual output folder of the tests: `TiffEncoder_EncodeColorPalette_WithPackBitsCompression_Works_Rgba32_Calliphora_palette_uncompressed.tiff`
> 
> Maybe we should add a CompareToReference output step in the tests to cover those cases.

@brianpopow here is my output, it seems it looks normal, maybe can be different results on different computers?
[TiffEncoder_EncodeColorPalette_WithPackBitsCompression_Works_Rgba32_Calliphora_palette_uncompressed.zip](https://github.com/SixLabors/ImageSharp/files/5975500/TiffEncoder_EncodeColorPalette_WithPackBitsCompression_Works_Rgba32_Calliphora_palette_uncompressed.zip)

I think adding CompareToReference is a good idea.